src/tutorial/src/step-2/description.md

```diff
@@ -23,58 +23,58 @@ console.log(counter.count) // 0
 counter.count++
 ```
 
-`reactive()` only works on objects (including arrays and built-in types like `Map` and `Set`). `ref()`, on the other hand, can take any value type and create an object that exposes the inner value under a `.value` property:
+`reactive()` funziona solo sugli oggetti (compresi gli array e i tipi nativi come `Map` e `Set`). `Ref`, invece può prendere qualsiasi tipo di valore e creare un oggetto che espone il suo valore interno sotto la proprietà `.value`.
 
 ```js
 import { ref } from 'vue'
 
-const message = ref('Hello World!')
+const message = ref('Ciao Mondo!')
 
-console.log(message.value) // "Hello World!"
-message.value = 'Changed'
+console.log(message.value) // "Ciao Mondo!"
+message.value = 'Modificato'
 ```
 
-Details on `reactive()` and `ref()` are discussed in <a target="_blank" href="/guide/essentials/reactivity-fundamentals.html">Guide - Reactivity Fundamentals</a>.
+Dettagli su `reactive()` e `ref()` sono discussi in <a target="_blank" href="/guide/essentials/reactivity-fundamentals.html">Guide - Le basi della Reattività</a>.
 
 <div class="sfc">
 
-Reactive state declared in the component's `<script setup>` block can be used directly in the template. This is how we can render dynamic text based on the value of the `counter` object and `message` ref, using mustaches syntax:
+Lo stato reattivo dichiarato nel blocco `<script setup>` del componente può essere usato direttamente nel template. Ecco come possiamo rendere un testo dinamico basato sul valore dell'oggetto `counter` e del ref `message`, usando la sintassi dei baffi:
 
 </div>
 
 <div class="html">
 
-The object being passed to `createApp()` is a Vue component. A component's state should be declared inside its `setup()` function, and returned using an object:
+L'oggetto passato a `createApp()` è un componente Vue. Lo stato di un componente deve essere dichiarato all'interno della sua funzione `setup()` ed essere restituito come un oggetto.
 
 ```js{2,5}
 setup() {
   const counter = reactive({ count: 0 })
-  const message = ref('Hello World!')
+  const message = ref('Ciao Mondo!')
   return {
     counter,
     message
   }
 }
 ```
 
-Properties in the returned object will be made available in the template. This is how we can render dynamic text based on the value of `message`, using mustaches syntax:
+Le proprietà che restituisce l'oggetto saranno rese disponibili nel template. Questo è il modo in cui possiamo rendere un testo dinamico basato sul valore `message`, usando la sintassi dei baffi:
 
 </div>
 
 ```vue-html
 <h1>{{ message }}</h1>
-<p>count is: {{ counter.count }}</p>
+<p>valore di count: {{ counter.count }}</p>
```

Comment:
in altri punti della guida, lo stesso esempio con  count è stato tradotto con: conteggio , possibilmente usiamo conteggio